app/src/main/res/values-fr/arrays.xml

```diff
@@ -1,14 +1,57 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string-array name="fps_names">
+        <item>30 IPS</item>
+        <item>60 IPS</item>
+        <item>90 IPS</item>
+        <item>120 IPS</item>
+    </string-array>
+    <string-array name="language_names" translatable="false">
```

Comment:
You should not translate the ones that say `translatable="false"`. Those should be left out of this file.

Reply:
Ok I didn't know sorry it's Okay I changed why not translate them ?

Reply:
They are not displayed to the user. They are used for internal purposes.